searx/settings.yml

```diff
@@ -434,6 +434,23 @@ engines:
     engine: archlinux
     shortcut: al
 
+  - name: nixos wiki
+    engine: mediawiki
+    shortcut: nixw
+    base_url: https://wiki.nixos.org/
+    search_type: text
+    disabled: true
+    categories: [it, software wikis]
+
+  - name: nixos wiki unofficial
+    engine: mediawiki
+    shortcut: nixwu
+    base_url: https://nixos.wiki/
+    search_type: text
+    disabled: true
+    categories: [it, software wikis]
+    api_path: api.php
```

Comment:
I myself wouldn't list this anymore. Most content got carried over to the official one and that is what is maintained by most people. So in the end you will likely get duplicated results.

Reply:
Yeah, fully understand. I was originally thinking of lowering the weight of the unofficial one. Just settled on giving it the slightly longer prefix, in case the official one doesn't have X or Y. I'm newer to this repo and to Nix, so feel free to remove it

Reply:
@SuperSandro2000 @Denperidge thanks for clarify .. I will remove the unofficial one .. we already have over 200 engines and should not add less maintained databases .. when there is an official one :-)

```yaml
engines:
  ...
  - name: nixos wiki unofficial
    engine: mediawiki
    shortcut: nixwu
    base_url: https://nixos.wiki/
    search_type: text
    disabled: true
    categories: [it, software wikis]
    api_path: api.php
```